src/generate-analysis.ts

```diff
@@ -295,7 +295,13 @@ function serializeElementLike(
       cssVariables: [],
       selectors: [],
     },
-    demos: (elementOrMixin.demos || []).map((d) => d.path),
+    demos: (elementOrMixin.demos || []).map((demo) => {
+      const json: Demo = {url: demo.path};
```

Comment:
I did this to prefer excluding the description key in the object when value was undefined.